backend/Makefile

```diff
@@ -19,10 +19,9 @@ db/connect:
 
 db/download:
     # Download the database to corpora_dev-<date>.sqlc
-	# Usage: make db/download FROM=dev
-	$(eval DATABASE_URI = $(shell aws secretsmanager get-secret-value --secret-id corpora/backend/${FROM}/database --region us-west-2 | jq -r '.SecretString | fromjson.database_uri'))
-	$(eval OUTFILE = $(shell date +corpora_${FROM}-%Y%m%d%H%M.sqlc))
-	pg_dump -Fc --dbname=$(DATABASE_URI) --file=$(OUTFILE)
+	$(eval DB_PW = $(shell aws secretsmanager get-secret-value --secret-id corpora/backend/${DEPLOYMENT_STAGE}/database --region us-west-2 | jq -r '.SecretString | split(":") | .[-1] | split("@") | .[0]'))
```

Comment:
so we still need to tunnel in before running this?

Reply:
Yeah